src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -825,7 +825,16 @@ private ImmutableArray<BoundExpression> BindInterpolatedStringParts(BoundUnconve
             var interpolatedStringConversion = memberAnalysisResult.ConversionForArg(interpolatedStringArgNum);
             Debug.Assert(interpolatedStringConversion.IsInterpolatedStringHandler);
             var interpolatedStringParameter = GetCorrespondingParameter(ref memberAnalysisResult, parameters, interpolatedStringArgNum);
-            Debug.Assert(interpolatedStringParameter.Type is NamedTypeSymbol { IsInterpolatedStringHandlerType: true });
+            Debug.Assert(interpolatedStringParameter is { Type: NamedTypeSymbol { IsInterpolatedStringHandlerType: true } }
+#pragma warning disable format
```

Comment:
nit: Do we need a tracking issue for formatting problem?

Reply:
@sharwell, do you know if there's a tracking issue for the formatter pulling this `or` all the way to the right, rather than leaving it aligned as it?